src/components/templates/PartyMap/components/Map/OldCampMap.scss

```diff
@@ -302,6 +302,7 @@ $border-radius: 28px;
     width: 100%;
```

Comment:
Need to test that nothing breaks when this is deleted. If it works without it, delete it.

Reply:
removed in https://github.com/sparkletown/sparkle/pull/840/commits/3fe0473136a7d3425fdfba216ef831b23f47552f